A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_PRESS.js

```diff
@@ -294,9 +298,28 @@ var A320_Neo_LowerECAM_PRESS;
 
             if (manMode && !this.oldManModValue) {
                 this.htmlMANtext.setAttribute("visibility", "visible");
+                this.htmlLdgElevText.setAttribute("visibility", "hidden");
+                this.htmlLdgElevValue.setAttribute("visibility", "hidden");
+                this.htmlLdgElevTitle.setAttribute("visibility", "hidden");
+                this.htmlLdgElevValueUnit.setAttribute("visibility", "hidden");
+                this.htmlSYS1text.setAttribute("visibility", "hidden");
+                this.htmlSYS2text.setAttribute("visibility", "hidden");
+                this.manModeTime = Date.now();
                 this.oldManModValue = 1;
             } else if (!manMode && this.oldManModValue) {
                 this.htmlMANtext.setAttribute("visibility", "hidden");
+                this.htmlLdgElevText.setAttribute("visibility", "visible");
+                this.htmlLdgElevValue.setAttribute("visibility", "visible");
+                this.htmlLdgElevTitle.setAttribute("visibility", "visible");
+                this.htmlLdgElevValueUnit.setAttribute("visibility", "visible");
+                // The switch from one system to the other should only happen if there's at least 10s between presses - placeholder until PRESS system is fully simulated
+                if ((Date.now() - this.manModeTime) > 10000) {
+                    activeSystem === 1 ? SimVar.SetSimVarValue("L:CPC_SYS1", "Bool", 0) : SimVar.SetSimVarValue("L:CPC_SYS2", "Bool", 1);
```

Comment:
> `"L:CPC_SYS1"` ?

is this a new variable

Reply:
No, it was already in the PRESS page code created in PR #1373 (line no. 220). I don't think it's being set or modified anywhere else at the moment though.